modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -242,7 +248,8 @@ sudo apt install phpmyadmin
 
 === Start a Service After a Resource is Mounted
 
-If you have network resources, such as NFS or iSCSI based mounts and you want to make sure that the database or web server only starts _after_ the resource is mounted.
+If you have network resources, such as NFS or iSCSI based mounts and you want to make
+sure that thedatabase or web server only starts _after_ the resource is mounted.
 Consider the following example setup when configuring your system.
```

Comment:
```suggestion
then consider the following example setup when configuring your system.
```
Maybe after doing this, then it will make sense?

Reply:
But it should start with a capital T (Then) - or?